account_invoice_tax_required/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import account_invoice
```

Comment:
```suggestion
from . import account_move
```
then change account_invoice.py to account_move.py